src/repl.rs

```diff
@@ -284,7 +294,12 @@ impl State {
             _ => format!("{current_database}"),
         };
 
-        let prompt = format!("{location}{txstate}> ");
+        let lang = match self.input_language {
+            InputLanguage::EdgeQL => "",
+            InputLanguage::SQL => "[sql]",
```

Comment:
Should we display a tag for EdgeQL too?

Reply:
SQL in CLI is marginal, I would not think that cluttering the default mode is worth it.